bcda/service/service.go

```diff
@@ -496,10 +496,15 @@ func (s *service) getBenesByFileID(ctx context.Context, cclfFileID uint, conditi
 			upperBound = time.Now()
 		}
 
-		ignoredMBIs, err = s.repository.GetSuppressedMBIs(ctx, s.sp.lookbackDays, upperBound)
-		if err != nil {
-			return nil, fmt.Errorf("failed to retreive suppressedMBIs %s", err.Error())
+		if cfg, ok := s.GetACOConfigForID(conditions.CMSID); ok {
+			if cfg.Model != "TCOCMD" {
```

Comment:
In `NewService` Im seeing:
```
return &service{
		...
		sp: suppressionParameters{
			includeSuppressedBeneficiaries: false,
			lookbackDays: cfg.SuppressionLookbackDays,
		},
		...
```
Is there a reason we cant use `includeSuppressedBeneficiaries` as opposed to a hardcode check of "TCOCMD"?

This means when we create the MD TCoC service (or link it to the ACO?) we will have to override the suppression parameters?

Reply:
yeah, [I saw that too](https://github.com/CMSgov/bcda-app/pull/993#discussion_r1765839218) and would much more prefer to use that but it's part of the service structure that's instantiated with our handlers (called on service start instead of per request).

Reply:
Ok so a service is instantiated on app start.  We pass something in as part of each request (ACO ID?) to get the specific request ACO out of all of the available ACOConfigs of the service.  It looks like we are pulling ACOConfigs from files in conf/config/{env}.yml?  Is there a reason we cant add onto these a new field eg `ignoreSuppressed`?

Somewhat related question how do these ACOConfigs compare with what we have in the acos DB table?  Looks like the DB table is just identifiers and web/auth related, no config?

Reply:
This has been updated to retrieve a setting from the config file!